templates/create-frontend-plugin/skeleton/plugins/${{values.plugin_id}}/catalog-info.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: ${{ values.plugin_id }}
+  {%- if values.description %}
+  description: ${{ values.description }}
+  {%- endif %}
+  annotations:
+    backstage.io/techdocs-ref: dir:.
+    github.com/project-slug: ${{ values.orgName }}/${{ values.repoName }}
+spec:
+  type: website
```

Comment:
I ended up just choosing one from this [list](https://backstage.io/docs/features/software-catalog/descriptor-format#spectype-required). I did it this way because the catalog page tends to be [pre-configured](https://github.com/backstage/backstage/blob/master/packages/app/src/components/catalog/EntityPage.tsx#L279) to include them and this way users outside of RHDH are still able to run the templates in their own instances of Backstage.

Reply:
Yeah, makes sense. This was just a `maybe-good-to-do` suggestion. Website / service works fine as well.